scripts/dockerhost/start-dockerhosts

```diff
@@ -80,25 +97,30 @@ first="true"
 for i in `seq 1 $num_nodes`; 
 do
     host="netplugin-node$i"
-    echo "Starting $host"
-    # godep modifies the host's GOPATH env variable, CONTIV_HOST_GOPATH
-    # contains the unmodified path passed from the Makefile, use that
-    # when it is defined.
-    if [ -n $CONTIV_HOST_GOPATH ]; then
```

Comment:
Hmm. maybe that got pruned during the merge. sorry for the distraction! Will fix this weekend.

> On May 22, 2015, at 2:23 PM, Madhav Puri <notifications@github.com> wrote:
> 
> In scripts/dockerhost/start-dockerhosts <https://github.com/contiv/netplugin/pull/66#discussion_r30931656>:
> 
> > @@ -80,25 +97,30 @@ first="true"
> >  for i in `seq 1 $num_nodes`; 
> >  do
> >      host="netplugin-node$i"
> > -    echo "Starting $host"
> > -    # godep modifies the host's GOPATH env variable, CONTIV_HOST_GOPATH
> > -    # contains the unmodified path passed from the Makefile, use that
> > -    # when it is defined.
> > -    if [ -n $CONTIV_HOST_GOPATH ]; then
> I think this change is needed to correctly mount the host's gopath.
> 
> The current sanity failure seems unrelated though.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/contiv/netplugin/pull/66/files#r30931656>.
>